CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+## Dradis Framework 3.11.1 (Dec, 2018)
```

Comment:
We're not shipping this as an individual release, please update to 3.12 here and in `gem_version.rb` then merge.